test/molecule-role/molecule/compose/prepare.yml

```diff
@@ -25,6 +25,12 @@
     - name: Add Docker GPG APT key
       become: true
       apt_key: url=https://download.docker.com/linux/ubuntu/gpg
+    - name: Set authorized key taken from file
```

Comment:
why do you need this ?

Reply:
For local runs, or ubuntu based runs - it will auto add key from your local notebook, for other cases - step will be ignored.

in that way ssh-ing to instance happens w/o need to lookup current onetime molecule key

Reply:
but what is the problem in using the ssh key which molecule generated?

Reply:
No specific problem. Recurrent minor time used to locate that key and inject via sshkey-add